test/specs.js

```diff
@@ -181,4 +181,30 @@ describe('L.Proj.CRS', function() {
 			.to.be.within(111.194, 111.195);
 
 	});
+
+	it('tests that _closestElement works (L.Proj.CRS._closestElement)', function testClosestElement() {
```

Comment:
While I definitely like that you've added a test, it is my opinion that we should test the class' external API, not its internals or exact implementation. Following this make tests less brittle to changes inside the class.

`_closestElement` is an implementation detail, and marked as such (prefixed by `_`) - it would be better to create a test emulating the state that caused it to break before your change: presumably by calling `zoom` with a value outside the resolution range.